### PR TITLE
Add number component to tests

### PR DESCRIPTION
**Pull request by cob16**

### CODE REVIEW

File: acceptance-tests/Gemfile.lock [hunk 1]
```diff
@@ -56,9 +72,12 @@ PLATFORMS
...
+  httparty
+  mail
+  pdf-reader
+  pry
   rspec
   selenium-webdriver
-  sinatra
 
 RUBY VERSION
    ruby 2.6.4p104
```
Comment (emileswarts) [inline acceptance-tests/Gemfile.lock]: Should there be a modified Gemfile in this PR?

File: acceptance-tests/spec/email_with_pdf_spec.rb [hunk 2]
```diff
@@ -25,6 +25,10 @@
     fill_in 'COMPOSITE.date-month', with: '11'
     fill_in 'COMPOSITE.date-year', with: '2007'
     continue
+
+    fill_in 'number_cats', with: 28
```
Comment (emileswarts) [inline acceptance-tests/spec/email_with_pdf_spec.rb]: That's a lot of cats

File: acceptance-tests/Gemfile.lock (hunk 1 above)
Comment (cob16) [reply acceptance-tests/Gemfile.lock]: this is the lockfile not the Gemfile. that likly meens the lock change was not committed while back so its just

Comment (cob16) [reply acceptance-tests/Gemfile.lock]: this is the lockfile `sinatra` was removed from the gems a while back.

**Pull request merged**